packages/relay-kit/src/packs/safe-4337/Safe4337Pack.ts

```diff
@@ -75,7 +75,7 @@ export class Safe4337Pack extends RelayKitBasePack<{
   #BUNDLER_URL: string
 
   #ENTRYPOINT_ADDRESS: string
-  #SAFE_4337_MODULE_ADDRESS: string = '0x'
+  #SAFE_4337_MODULE_ADDRESS: Hash = '0x'
```

Comment:
I changed it back to string on my branch. Maybe something went wrong when merging the latest changes from my branch into yours 🤔

Reply:
Actually, I did revert all these changes on the rely-kit